File: ruby/fibonacci.rb
#!/usr/bin/ruby
#
#  Counts the fibonacci sequence
#  ruby fibonacci.rb
#


def generateFibonacci(sPos =1, ePos =100)

  stack = [sPos];
  for i in sPos..ePos
    stackCount = stack.length
    if stackCount <= 2
      stack.push(1)
    else
      stack.push(stack[stack.length - 1] + stack[stack.length - 2])
    end
    print "\n #{stack[i]}"

  end # end for loop

    return stack

end


generateFibonacci()

File: python/fibonacci.py
#!/usr/bin/python
#
#  Counts the fibonacci sequence
#  ruby fibonacci.rb
#

def generateFibonacci(sPos = 1, ePos = 100):

  stack = [sPos]
  for i in range(sPos,ePos):
    stackCount = len(stack)
    if (stackCount <= 2):
      stack.append(1)
    else:
      value = stack[len(stack) - 1] + stack[len(stack) - 2]
      stack.append(value)
    print  str(stack[i])

  return stack

generateFibonacci()
